crowdsourcehinter/static/html/crowdsourcehinterstudio.html

```diff
@@ -15,7 +15,7 @@
         This is the element of the problem for which the hinter is working. This is the "data-usage-id" of the problem block. It should look something like i4x://edX/DemoX/problem/f958789435cf47218ff32f0d600f1184
     </p>
     <p>
-        Problem Element: <textarea type="text" class="csh_hinting_element" value = "{{hinting_element}}"/>
+        Target Problem: <textarea type="text" class="csh_target_problem" value = "{{target_problem}}"/>
```

Comment:
Doesn't need to be resolved here and now, but better guidance to the user on what this is would be helpful.